bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/types/IntersectableReferenceType.java

```diff
@@ -25,6 +25,7 @@
  *
  * @since 2.0.0
  */
+@Deprecated
```

Comment:
Can we remove this next update?

Reply:
We usually deprecate the API right. We can keep this in the next update as well?